src/lib.rs

```diff
@@ -38,9 +38,9 @@ macro_rules! assert_eq_hex {
                     // The reborrows below are intentional. Without them, the stack slot for the
                     // borrow is initialized even before the values are compared, leading to a
                     // noticeable slow down.
-                    panic!(r#"assertion failed: `(left == right)`
-  left: `{:#x?}`,
- right: `{:#x?}`: {}"#, &*left_val, &*right_val,
+                    panic!(r#"assertion `left == right` failed
+  left: {:#x?},
+ right: {:#x?}: {}"#, &*left_val, &*right_val,
```

Comment:
this is not what the "modern rust" will have for assert_eq.  The user context string is printed on the first line after the `failed: {...}`

Reply:
True!, opened https://github.com/wcampbell0x2a/assert_hex/pull/5. I think this follows all the way.